client/packages/common/src/intl/locales/en/common.json

```diff
@@ -164,14 +166,31 @@
   "label.unit-price": "Unit Price",
   "label.unit-quantity": "Unit Qty",
   "label.units": "Units",
+  "label.user": "User",
   "label.verified": "Verified",
   "label.website": "Website",
   "link.copy-to-clipboard": "Copy to Clipboard",
+  "log.invoice-created": "Shipment created",
```

Comment:
with a fallback to the SHOUTY_UNDERSCORE_CAPS version. translating `invoice` to `shipment` is nice, I think and worth the effort.

Reply:
How is the fallback working? e.g. you mean if there is a new enum that is not in the here?

Reply:
yes, just that. If there is a new enum value, then the translation will show the string value of the enum rather than a translation.